src/utils/transactions/SafeTransaction.ts

```diff
@@ -5,10 +5,14 @@ export enum OperationType {
   DelegateCall // 1
 }
 
-export interface SafeTransactionData {
+export interface SafeBasicTransactionData {
```

Comment:
No super strong opinion but I would probably prefer a naming like:

`SafeTransactionData` or `MetaTransactionData` (for the base)
and `MultisigSafeTransactionData` or similar ... but again ... probably not worth spending too much time on as none of them feel perfect :P

Reply:
Yes, I wasn't convinced with the name I used either.
As you said, I think I like `MultisigSafeTransactionData` and `MultisigSafeTransactionDataPartial` for the types that contain all the params (required and optional respectively) and `MetaTransactionData` for the one that only contains the parameters needed in the transactions inside the MultiSend.